php/src/Constant.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * 常用的常數
+ *
+ * @version 0.1.0
+ * @author eden.chen eden.chen@kkday.com
+ * @date 2024/12/4
+ * @since 0.1.0 2024/12/4 eden.chen: 新建立Constant class
+ */
+
+namespace GildedRose;
+
+/**
+ * Constant Class
+ */
+class Constant
+{
+    /**
+     *  品質最大值
+     */
+    const MAX_QUALITY = 50;
```

Comment:
`[EA] 'MAX_QUALITY' should be declared with access modifier. `

`public const MAX_QUALITY = 50;`